### PR TITLE
Add xxhash and storage key test for Kusama

### LINKED ISSUE #800: xxHash support
http://cyan4973.github.io/xxHash/

### PR DESCRIPTION
**Pull request by hewigovens**
Fixes #800

### CODE REVIEW

File: src/interface/TWHash.cpp [hunk 1]
```diff
@@ -6,86 +6,88 @@
 
 #include "../Data.h"
 
-#include <TrustWalletCore/TWHash.h>
+#include "BinaryCoding.h"
+#include "xxhash64.h"
```
Comment (catenocrypt) [inline src/interface/TWHash.cpp]: Should we name the file XXHash64.h (Pascal-case, as for other files).

File: src/xxhash64.h [hunk 2]
```diff
@@ -0,0 +1,203 @@
+// Copyright © 2016 Stephan Brumme. All rights reserved, see http://create.stephan-brumme.com/disclaimer.html
```
Comment (catenocrypt) [inline src/xxhash64.h]: Also add link to: https://github.com/Cyan4973/xxHash/blob/dev/LICENSE
Their BSD clause requires this, for source redistribution.

File: src/interface/TWHash.cpp (hunk 1 above)
Comment (hewigovens) [reply src/interface/TWHash.cpp]: done

File: src/xxhash64.h (hunk 2 above)
Comment (hewigovens) [reply src/xxhash64.h]: This code is not from Cyan4973/xxHash

**Pull request merged**